### PR TITLE
Use adobe opensource workflow icons and fix #60

### LINKED ISSUE #60: Icons broken due to missing 'spectrum-icons.svg'
### Expected Behavior
Icons show up in the proper `html` when the `loadIcons()` function is called, passing as the parameter the path of e.g. the `'spectrum-css-icons.svg'` or `'spectrum-css-icons-medium.svg'` (as suggested in the doc)

### Actual Behavior
They don't. Actually, running Gulp in the current repo produces a `dist/docs` page where the icons are missing too. As a side note, the readme mentions `AdobeSpectrum.loadIcons()`, while the currently provided function is just `loadIcons()`

### Reproduce Scenario (including but not limited to)
Run Gulp and open `dist/docs` – the only icon that works (e.g. in the Action Button) is the one where the `svg` is passed literaly.

#### Spectrum-CSS version
2.7.2

#### Sample Code that illustrates the problem
```
<button class="spectrum-ActionButton">
    <svg class="spectrum-Icon spectrum-Icon--sizeS spectrum--medium" focusable="false" aria-hidden="true"
        aria-label="Edit">
        <use xlink:href="#spectrum-icon-18-Edit"></use>
    </svg>
    <span class="spectrum-ActionButton-label">Edit</span>
</button>
```
#### Workaround

Inspecting the way the online doc page works (as opposed to the repo's gulp generated doc), the file that is passed to `loadIcons()` is **`'spectrum-icons.svg'`**, and not `'spectrum-css-icons.svg'`. Problem is that such file is not found anywhere in the repository.

Dowloading it from the online page and using it locally, icons finally do display!

Thanks,

Davide

### PR DESCRIPTION
**Pull request by jianliao**
## Description
Switch from private @spectrum/spectrum-icons package to open sourced @adobe/spectrum-css-workflow-icons package.

## Related Issue
#60 

## Motivation and Context
1. Help contributor to get all the Adobe open sourced workflow icons on their local box.
2. Fully enable visual regression testing.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by jianliao** [comment]
Not sure why this PR still includes all the commits from lerna branch, but I will squash and merge to make sure it would not go into commit history.

File: bundles/spectrum-css-compat/package.json [hunk 1]
```diff
@@ -20,10 +20,8 @@
   "bugs": {
     "url": "https://github.com/adobe/spectrum-css/issues"
   },
-  "optionalDependencies": {
-    "@spectrum/spectrum-icons": "^2.3.0"
-  },
   "devDependencies": {
+    "@adobe/spectrum-css-workflow-icons": "^1.0.0-beta.3",
```
Comment (bernhardsf) [inline bundles/spectrum-css-compat/package.json]: very nice!

Comment (bernhard-adobe) [inline bundles/spectrum-css-compat/package.json]: very nice!

**Comment by jianliao** [comment]
Close as I created #261 with clear commit history.

**Pull request closed without merging**